anyway/infographics_utils.py

```diff
@@ -5,12 +5,12 @@
 import os
 from functools import lru_cache
 
+import gettext
```

Comment:
1. `from gettext import gettext as _`
2. why arey you replacing `flask_babel` with another library?

Reply:
That's correct, sorry about that.